tapgarden/interface.go

```diff
@@ -43,6 +43,14 @@ type Planter interface {
 	// returned.
 	CancelSeedling() error
 
+	// FundBatch attempts to provide a genesis point for the current batch,
+	// or create a new funded batch.
+	FundBatch(params FundParams) (*MintingBatch, error)
```

Comment:
So with the the _user_ will select the genesis point? I thought it was just enough to do the coin selection earlier, then make it available to the user if needed so they can provide the custom witness?

Reply:
The `FundBatch` call, in this PR, just triggers coin selection, yes. The `FundParams` are the same as `FinalizeParams`. In some future PR we can expand those params to include a template PSBT with a genesis point (which would address #721)